tutorials/users/ide/version-control.md

```diff
@@ -1 +1,614 @@
+
+When done click the `Commit` button below the commit description.  If successful you will see a `Committed changes` status pop up in the lower right corner.
+
+![Git Commit Successful](images/version-control/git-commit-successful.png)
+
+### Pushing a Commit
+
+Making a commit sets a "flag" in the progress of your work that shows what has been done at certain points. Pushing a commit moves these changes to the remote 
+repository (if you've also [setup a remote repository](adding_remote_repo)).
+
+Click the `Git` menu and `Push to Remote`
+
+![Git Push Menu](images/version-control/git-push-menu.png)
+
+Enter in your login credentials.  For the password enter your Personal Access Token.  Click `OK`.
+
+
+![Git Push Credentials](images/version-control/git-push-credentials.png)
+
+If successful you will see a `Successfully pushed` status pop up in the lower right corner.
+
+![Push Successful](images/version-control/push-successful.png)
+
+
+
+### View Commit History
+To see all your past commits click the `History` tab in the `Commits` section under `Current Branch`
+
+Each commit is identified with the name of the user who made it and the commit tag. A commit tag is the first few characters
+of a unique cryptographic hash of your commit.
+
+In the example below, the commit tag of the first commit is `01f1dc4`.  The name of the user has been removed from 
+the image to protect privacy.
+
+![Commit History](images/version-control/git-log-gui.png)
+
+
+
+
+### View the Changes Made to a File Between Versions
+
+There are several ways to view the changes made between versions of a file.
+
+1. Right-click on the file and select `Diff`.  
+
+   ![Diff Right Click](images/version-control/right-click-menu.png)
+
+2. Click the `Diff` button.
+
+   ![Diff Button](images/version-control/diff-button.png)
+
+3. Enable the double-click shortcut for viewing a `Diff`.  
+
+   From the `Git` menu select **"Double click opens diff"**.  A checkmark will appear to show this option is enabled. 
+Then double-click on the file.
+
+   ![Diff Double Click](images/version-control/git-menu-diff.png)
+
+The `diff` screen of the file will open and show the changes made to the file.  On the left is the `HEAD` panel 
+and on the right is the `WORKING` panel. 
+
+`HEAD` is the most current version of the file without the current changes you made.  Areas where changes were made are 
+highlighted in red.  If additions were made there will be green slashes on top of the red highlight.  
+If there were deletions they will be highlighted in darker red on top of the red highlight. 
+
+`WORKING` is the new version of the file.  Areas where changes were made are 
+highlighted in green.  
+
+![Diff Panel](images/version-control/diff-panel.png)
+
+
+### Undoing Changes
+Undoing changes can mean several things:
+
+1. `Discard` - Undo changes made to the current unstaged file
+
+   An unstaged file means this file has been changed since the last committed version.  
```

Comment:
This also describes a staged file. I'd also add to this sentence:

"and has not yet been staged"